plugins/nvfetcher.toml

```diff
@@ -1,11 +1,3 @@
-[intellitab-nvim]
-src.git = "https://github.com/pta2002/intellitab.nvim"
-fetch.github = "pta2002/intellitab.nvim"
-
-[mark-radar]
-src.git = "https://github.com/winston0410/mark-radar.nvim"
-fetch.github = "winston0410/mark-radar.nvim"
-
 [coq-nvim]
 src.git = "https://github.com/ms-jpq/coq_nvim"
 fetch.github = "ms-jpq/coq_nvim"
```

Comment:
We could delete this and rely on `overrideAttrs` with the upstream package right ?